package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@blackbaud/auth-client",
-  "version": "2.24.0",
+  "version": "2.25.0",
```

Comment:
Since you bumped the version number already, could you also bump it at the top of `package-lock.json` and add an entry to `CHANGELOG.md` along these lines (you can thank yourself) :):

"Added additional domain to the auth domain whitelist. #136 (Thanks @Blackbaud-JackieKirchhoff!)"